tools/doc/generate.js

```diff
@@ -61,33 +61,37 @@ args.forEach((arg) => {
 nodeVersion = nodeVersion || process.version;
 
 if (!filename) {
-  throw new Error('No input file specified');
+  throw new Error("No input file specified");
 } else if (!outputDir) {
-  throw new Error('No output directory specified');
+  throw new Error("No output directory specified");
 }
 
+const fileTask = async () => {
+  try {
```

Comment:
instead of wrapping this in a try/catch, add a catch handler to your call to fileTask.